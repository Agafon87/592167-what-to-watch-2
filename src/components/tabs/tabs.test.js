import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs";

it(`Tabs correctly renderer after test`, () => {
  const tree = renderer.create(<Tabs film={{}}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
