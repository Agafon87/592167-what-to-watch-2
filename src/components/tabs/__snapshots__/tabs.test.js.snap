// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs correctly renderer after test 1`] = `null`;
